### DISCUSSION TITLE
After installing using TrollStore, there is no option in the settings.

### DISCUSSION THREAD

**Discussion by MoeCinnamo** [utmapp/UTM#6065] [UTM Help (iOS)]
**Describe the issue**
After installing using TrollStore, there is no option in the settings.

**Configuration (required)**
* UTM Version: 4.4.5
* OS Version: iPadOS 16.6
* Device Model: A1934
* Is it jailbroken (name jailbreak used)? No
* How did you install UTM? Use TrollStore 2.0.12

**Crash log**
No log

**Debug log**
No log

**Upload VM**
No VM

**Comment by MoeCinnamo** [comment]
![IMG_0333](https://github.com/utmapp/UTM/assets/101342651/086abbb5-547f-4e52-a070-a4cd262978f0)
![IMG_0334](https://github.com/utmapp/UTM/assets/101342651/aacc802a-3453-42b9-a1a6-32ba306f4f62)
This is a recurrence.

**Reply by bewl87** [reply]
just switch to user registration in trollstore if you need to see in settings/icloud etc ![image](https://github.com/utmapp/UTM/assets/49694850/a8220a05-4472-4022-b703-2ffcf863275f)

**Comment by osy** [comment]
This is a troll store issue. Use the in-app settings.
